core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/store/dot-palette.store.ts

```diff
@@ -183,7 +190,6 @@ export class DotPaletteStore extends ComponentStore<DotPaletteState> {
                     .getWithOffset((event && event.first) || 0)
                     .pipe(take(1))
                     .subscribe((data: DotCMSContentlet[] | DotCMSContentType[]) => {
-                        data.forEach((item) => (item.contentType = item.variable = 'FORM'));
                         this.setLoaded();
                         this.setContentlets(data);
                         this.setTotalRecords(this.paginationService.totalRecords);
```

Comment:
I know this code wasn't modified but we are mutating here three times the `state`. It would be better to have one updater to set these three properties at once.

Reply:
Yes, we need to improve this store heavily, but making those changes is out of the scope of the issue.